harness/determined/deploy/aws/injector/master.yaml.tmpl

```diff
@@ -58,6 +58,9 @@ security:
 {{ end }}
 
 {{ if ne .lore_version "" }}
+feature_switches:
+  - GAS
```

Comment:
Could we call this GENAI?

Reply:
I am not familiar with feature_switches, could you explain what it does? Is it like a feature flag that can be turned on and off?

Reply:
I don't see much documentation on it. 
```
(determined) hbp ➜ determined git:(main) ag 'feature.?switch' proto -C 2 -G .proto
proto/src/determined/api/v1/master.proto
70-  Product product = 11;
71-  // List of features that is on.
72:  repeated string feature_switches = 12;
73-  // Feature flag for user management.
74-  bool user_management_enabled = 13;
(determined) hbp ➜ determined git:(main) ag 'feature.?switch' docs
(determined) hbp ➜ determined git:(main) ag 'feature.?switch' master/internal
master/internal/api_master.go
48:		FeatureSwitches:       a.m.config.FeatureSwitches,

master/internal/config/config.go
131:		FeatureSwitches: []string{},
159:	FeatureSwitches       []string                          `json:"feature_switches"`
(determined) hbp ➜ determined git:(main)
```

it was added by the web team for web use it looks like cc @gt2345 
```
commit 45d30bcaa5f8558085835adaa53f962215417777
  Author: Guangqing Tang <40620519+gt2345@users.noreply.github.com>
  Date:   Mon Oct 24 16:24:36 2022 -0500

      feat: implement feature switch [WEB-535] (#5310)
```

Reply:
I would think that "genai" is probably for the best since that's the pattern we were already following

Reply:
@hamidzr Could you please change the GAS reference to GENAI (both in the template and front-end). Otherwise, it looks good to me. Thanks!